src/tasks/register-authorizers.js

```diff
@@ -65,7 +65,8 @@ module.exports = function registerAuthorizers(authorizerMap, apiId, awsRegion, f
 					identitySource: identitySource,
 					name: authName,
 					restApiId: apiId,
-					type: type
+					type: type,
+					authorizerResultTtlInSeconds: 300
```

Comment:
if this is default, let's remove it from here and just let it use the default functionality. that way if they change the default in the future we don't need to adjust

Reply:
In case of removing the explicit default value then, of course, don't need to adjust but this action rises a UX problem.

The user will navigate to AWS dashboard and will see the value has been set to `undefined` and might assume that the cache is disabled. However, the cache is actually active for 300 seconds. I suppose AWS code handles undefined as an invalid value and returns the default value.